http-clients/src/test/java/org/triplea/http/client/moderator/toolbox/ModeratorToolboxClientTest.java

```diff
@@ -0,0 +1,158 @@
+package org.triplea.http.client.moderator.toolbox;
+
+import static java.util.Collections.singletonList;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.StringContains.containsString;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.hamcrest.core.IsSame;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.triplea.http.client.HttpInteractionException;
+
+@ExtendWith(MockitoExtension.class)
+class ModeratorToolboxClientTest {
+
+  private static final String TEST_VALUE = "test-value";
+  private static final String RETURN_VALUE = "return-value";
+  private static final String EXCEPTION_MESSAGE = "exception-message";
+  private static final HttpInteractionException EXCEPTION = new HttpInteractionException(500, EXCEPTION_MESSAGE);
+  private static final List<String> BAD_WORDS = Arrays.asList("word1", "word2");
+
+  private static final int ROW_COUNT = 5;
+  private static final int ROW_START = 10;
+
+  private static final String API_KEY = "api-key";
+  private static final Map<String, Object> expectedHeader;
+
+  static {
+    expectedHeader = new HashMap<>();
+    expectedHeader.put(ModeratorToolboxClient.MODERATOR_API_KEY_HEADER, API_KEY);
+  }
```

Comment:
Is there something speaking against `Collections#singletonMap`?

Reply:
Not initially, but it looks like 'API_KEY_PASSWORD' is going to be a second element soon.